frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx

```diff
@@ -102,7 +111,10 @@ const InventoryItem: React.FC<InventoryItemProps> = React.memo(
         </AccordionItem>
       </Accordion>
     ) : (
-      <div className="co-inventory-card__item">
+      <div
+        className="co-inventory-card__item co-inventory-card__item--border"
```

Comment:
The second class should not be there

Reply:
Why not? I didn't add it

Reply:
This diff clearly shows you did :) It is a result of an incorrect rebase. The `co-inventory-card__item--border` class was recently removed by another patch and when you rebased, it is being unintentionally added back.

Reply:
Fixed. Thanks